src/main/java/decodes/sql/PlatformListIO.java

```diff
@@ -288,21 +233,20 @@ public void read(PlatformList platformList)
 	protected void readAllTransportMedia(PlatformList platformList)
 		throws SQLException, DatabaseException
 	{
-		try (Statement stmt = createStatement();)
+		try (Statement stmt = createStatement())
```

Comment:
you can use doQuery here, it will loop through the result set list and call the provided function for each returned row.